src/App.tsx

```diff
@@ -1,27 +1,139 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
+import {
+  Container,
+  Stack,
+  Button,
+  Typography,
+} from '@mui/material';
+
 import { GoodsList } from './GoodsList';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+import { getAll, get5First, getRedGoods } from './api/goods';
+import { Good } from './types/Good';
+
+export const App: React.FC = () => {
+  const [goods, setGoods] = useState<Good[]>([]);
+
+  enum ButtonVariant {
+    outlined = 'outlined',
+    contained = 'contained',
+  }
+
+  const [buttonVariants, setButtonVariants] = useState({
+    all: ButtonVariant.outlined,
+    firstFive: ButtonVariant.outlined,
+    red: ButtonVariant.outlined,
```

Comment:
I think it would be really cool if we create an initial state variable and then reuse it on lines 53, 33, and 73 of your code.